verify-sna/assets/data/operations.private.js

```diff
@@ -1,120 +1,88 @@
 const assets = Runtime.getAssets();
-const {
-  getVerificationQuery,
-  updateVerificationQuery,
-  insertVerificationQuery,
-  deleteVerificationsQuery,
-  getAllVerificationsQuery,
-} = require(assets['/data/config.js'].path);
+const { PENDING_STATUS } = require(assets['/services/constants.js'].path);
 
-const run = async (db, query, params) => {
-  return new Promise((resolve, reject) => {
-    db.run(query, params, (err) => {
-      if (err) {
-        return reject(err);
-      }
-      return resolve();
-    });
-  });
-};
-
-const all = async (db, query, params) => {
-  return new Promise((resolve, reject) => {
-    db.all(query, params, (err, rows) => {
-      if (err) {
-        return reject(err);
-      }
-      return resolve(rows);
-    });
-  });
-};
-
-const get = async (db, query, params) => {
-  return new Promise((resolve, reject) => {
-    db.get(query, params, (err, row) => {
-      if (err) {
-        return reject(err);
-      }
-      return resolve(row);
-    });
-  });
-};
-
-const deleteVerifications = async (db) => {
+const getAllVerifications = async (syncMap) => {
   return new Promise(async (resolve, reject) => {
```

Comment:
Why do we need to make a promise something that is already that? I didn't notice this from the first pass, but I think it is redundant. It was required wit with the SQLite approach because it was all callback based.

Reply:
In the case of getAllVerifications, it's because I want to catch any error when fetching a verification to resolve the promise instead of rejecting it. In all the other cases it's in fact a redundancy and I already removed it. Thank you for pointing it out.

Reply:
Cool, thanks for checking it. Quick question, did we apply this in any other place where is applicable outside this operations file?

Reply:
I could also catch the error on the other side and work with it though. I don't know if that's what you meant.

Reply:
No, I mean in other files like [here](https://github.com/elkinnarvaez/function-templates/pull/4/files#diff-a525f8d4371de944f0966d5a38a236e2b2e1fa2f87049f2215df473b63706f70R1)

Reply:
Okay, I think all the redundancies are removed now. Thanks.